### PR TITLE
Use existing users rather than create them

### PR DESCRIPTION
**Pull request by dav3r**
#  #

## 🗣 Description ##
This PR updates this repository so that it:
* Manages users that have already been created elsewhere (see [`cisagov/cool-users-non-admin`](https://github.com/cisagov/cool-users-non-admin)) - the users are no longer created by this repository
* Is similar in style and convention to [`cisagov/cool-auditor-iam`](https://github.com/cisagov/cool-auditor-iam)

## 💭 Motivation and context ##
Now that all non-admin users are created in [`cisagov/cool-users-non-admin`](https://github.com/cisagov/cool-users-non-admin), we must update all of our repos related to these non-admin users accordingly.  These changes are very similar to those in https://github.com/cisagov/cool-auditor-iam/pull/1.

This PR is part of the work needed for https://github.com/cisagov/cool-system/issues/114.

Also note that this is the first PR after this repository was renamed from `cool-users-pca` to `cool-pca-iam` (hence the backend key renaming in https://github.com/cisagov/cool-pca-iam/pull/7/commits/a7954c0b17adee8c7f10488ce8609bfebee7b716).

## 🧪 Testing ##
I first removed some items from the Terraform state that were no longer part of this repository and that I didn't want Terraform to delete (these resources were previously imported to and are now owned by [`cisagov/cool-users-non-admin`](https://github.com/cisagov/cool-users-non-admin)):
```
terraform state rm aws_iam_user.users[\"pca-user-1\"] aws_iam_user.users[\"pca-user-2\"] aws_iam_user.users[\"pca-user-3\"]
terraform state rm aws_iam_user_policy.self_managed_creds_with_mfa[\"pca-user-1\"] aws_iam_user_policy.self_managed_creds_with_mfa[\"pca-user-2\"]
# Note that pca-user-3 did not have a policy to manage their own creds
```

Next, I ran a `terraform apply` and confirmed that no other changes were needed (as expected).

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by hillaryj — APPROVED** [review]
🌮

**Review by mcdonnnj — APPROVED** [review]
Makes sense. 🧹✔

File: variables.tf [hunk 1]
```diff
@@ -10,8 +10,8 @@ variable "pca_account_ids" {
 }
 
 variable "users" {
-  type        = map
-  description = "A map containing the usernames of each PCA user, a list of roles assigned to that user, and a boolean flag (\"self_admin\") that specifies if the user is allowed to self-administer their security credentials.  The currently-supported roles are: [\"provisioner\"].  Example: { \"firstname1.lastname1\" = { \"roles\" = [ \"provisioner\" ], \"self_admin\" = true },  \"firstname2.lastname2\" = { \"roles\" = [ \"provisioner\" ], \"self_admin\" = false } }"
+  type        = map(map(list(string)))
```
Comment (jsf9k) [inline variables.tf:13]: :fire:

**Review by jsf9k — APPROVED** [review]
Great work!  Thanks for picking up on the `map(map(list(string)))` thang.

**Pull request merged**